src/Adapters/Laravel/Commands/TestCommand.php

```diff
@@ -132,20 +133,67 @@ protected function phpunitArguments($options)
     protected function clearEnv()
     {
         if (!$this->option('env')) {
-            $repositories = RepositoryBuilder::create()
-                ->make();
-
-            $envs = Dotenv::create(
-                $repositories,
+            $vars = self::getEnvironmentVariables(
                 // @phpstan-ignore-next-line
                 $this->laravel->environmentPath(),
                 // @phpstan-ignore-next-line
                 $this->laravel->environmentFile()
-            )->safeLoad();
+            );
+
+            $repository = Env::getRepository();
 
-            foreach (array_keys($envs) as $name) {
-                $repositories->clear($name);
+            foreach ($vars as $name) {
+                $repository->clear($name);
             }
         }
     }
+
+    private function getEnvironmentVariables($path, $file)
+    {
+        if (class_exists(\Dotenv\Parser\Parser::class)) {
+            return self::getEnvironmentVariablesV5($path, $file);
+        }
+
+        if (class_exists(\Dotenv\Loader\Parser::class)) {
```

Comment:
Collision 5 will be Laravel 8 only. Can we remove all the code related to Dotenv@4?

Reply:
Also added this https://github.com/nunomaduro/collision/pull/126. Let me know if sounds good for you.

Reply:
I'm a bit confused. Which branch is for Laravel 8?

Reply:
No problem. The current `stable` branch contains Collision 5@beta. Collision 5 will ship in Laravel 8.